cmd/minikube/cmd/config/validations_test.go

```diff
@@ -0,0 +1,84 @@
+package config
```

Comment:
I think all these new files will need copyright headers. Travis should fail without them.